inference-engine/tests/functional/plugin/shared/src/execution_graph_tests/num_inputs_fusing_bin_conv.cpp

```diff
@@ -14,7 +14,7 @@
 
 #include "execution_graph_tests/num_inputs_fusing_bin_conv.hpp"
 
-#include "network_serializer.h"
+std::vector<InferenceEngine::CNNLayerPtr> TopologicalSort(const InferenceEngine::ICNNNetwork& network);
```

Comment:
Do we need to add extern?

Reply:
no, functions declaration is `extern` by default.